docs/release-process.md

```diff
@@ -1,45 +1,68 @@
-# Releases
+# Release Process
 
-NGINX Kubernetes Gateway uses semantic versioning for its releases. For more information see https://semver.org.
+This document outlines the release process for NGINX Kubernetes Gateway (NKG).
 
-Warning: Major version zero (0.y.z) is reserved for development, anything MAY change at any time. The public API is not stable.
+## Versioning
 
-NGINX Kubernetes Gateway, especially in its early stages, will release periodically on a feature set cadence. By feature set, we intend to mean 2-4 features as tracked by the repository's Github milestones. We do this to remain agile and to expose features relatively quickly, but also to minimize the cost of many frequent releases and the associated process overhead.
+NKG uses semantic versioning for its releases. For more information, see https://semver.org.
 
-This means users can expect a release generally every 1 or 2 months, and can also track the progress of a release and its features by viewing our Github milestones. Each issue scheduled for a release will be labelled with "enhancement" (initial feature requests are "proposals", the label "enhancement" will be exchanged when work is committed) and added to the milestone release, milestones will use the form vMAJOR.MINOR.PATCH. Multiple issues may be written to describe one use-case and each will be organized according to this process. Sub-issues should also link to their parent (the parent issue will list the links via mention notifications).
+> Major version zero `(0.Y.Z)` is reserved for development, anything MAY change at any time. The public API is not stable.
 
-Bugs and other issue types will use a similar format; once committed to a release they will be assigned to the milestone representing that release.
+## Release Planning and Development
 
-When all milestone work is complete, we will begin our release process and publish artifacts accordingly.
+Until NKG reaches version 1.0.0, we aim to release approximately every two months. This enables us to introduce new
```

Comment:
Why specify a release cadence?

Reply:
we can remove it if it doesn't add value or misleading. I figure it'd be useful information for users to know the release expectancy.

Reply:
I would vote to wait on adding this information until we have a few more releases under our belt so we better understand our natural cadence.